MapboxNavigation/UserHaloCourseView.swift

```diff
@@ -0,0 +1,110 @@
+import UIKit
+
+/**
+A view representing the user’s reduced accuracy location on screen.
+*/
+
+public class UserHaloCourseView: UIView, CourseUpdatable {
```

Comment:
Since `UserHaloCourseView` is now part of public API please update `jazzy.yml` to contain new symbol.